vendor/github.com/containerd/cgroups/v3/cgroup2/manager.go

```diff
@@ -242,8 +242,10 @@ func setResources(path string, resources *Resources) error {
 type CgroupType string
 
 const (
-	Domain   CgroupType = "domain"
-	Threaded CgroupType = "threaded"
+	Domain         CgroupType = "domain"
+	DomainThreaded CgroupType = "domain threaded"
+	DomainInvalid  CgroupType = "domain invalid"
+	Threaded       CgroupType = "threaded"
```

Comment:
Wondering if this fix is missing other changes; see my comment on the PR;

- https://github.com/containerd/cgroups/pull/324#discussion_r1869577003
- related to https://github.com/containerd/cgroups/pull/318

Reply:
- ☝️ PR open for this one https://github.com/containerd/cgroups/pull/358